build/yaml/templates/display-env-vars-steps.yml

```diff
@@ -0,0 +1,3 @@
+steps:
+- powershell: 'gci env:* | sort-object name | Format-Table -AutoSize -Wrap'
+  displayName: 'Display env vars'
```

Comment:
Not seeing where this is being used. Is it needed?

Reply:
No this was for debugging, will remove for now